.github/workflows/tests.yml

```diff
@@ -46,6 +47,10 @@ jobs:
           pip install -r requirements-ci.txt
           pip install wheel pytest pytest-cov pytest-mock flaky --upgrade
 
+      - name: Conditionally install openfermionpyscf
+        if: contains(' 3.9', matrix.config.python-version)
+        run: pip3 install openfermionpyscf

```

Comment:
So with this change, only the Python 3.9 tests are running the PySCF tests?

Reply:
Yes, I limited the pyscf tests to Python 3.9 to not further extend the testing for other core-tests specially the one with Python 3.8. I can add the 3.7 and 3.8 if it makes more sense.

Reply:
I think just 3.9 that's fine :+1: